apps/extension/src/ui/apps/dashboard/routes/Settings/Mnemonics/MnemonicBackupModal/Show/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { MnemonicUnlock } from "@ui/domains/Mnemonic/MnemonicUnlock"
+import { useMemo, useState } from "react"
+import { useTranslation } from "react-i18next"
+
+import { Stages, useMnemonicBackupModal } from "../context"
+import { MnemonicBackupModalBase } from "../MnemonicBackupModalBase"
+import { Verify } from "./Verify"
+import { ViewMnemonic } from "./View"
+
+enum ShowMnemonicStages {
+  View = "View",
+  Verify = "Verify",
+}
+
+export const ShowMnemonic = () => {
+  const { t } = useTranslation("admin")
+  const { mnemonic, setStage } = useMnemonicBackupModal()
+  const [showMnemonicStage, setShowMnemonicStage] = useState<ShowMnemonicStages>(
+    ShowMnemonicStages.View
+  )
+  const title = useMemo(() => {
+    switch (showMnemonicStage) {
+      case ShowMnemonicStages.View:
+        return t("Backup Recovery Phrase")
+      case ShowMnemonicStages.Verify:
+        return t("Verify Recovery Phrase")
+    }
+  }, [showMnemonicStage, t])
+
+  if (!mnemonic)
+    return <MnemonicBackupModalBase title={"Error"}>No mnemonic available</MnemonicBackupModalBase>
```

Comment:
Should this be `t("No mnemonic available")`? Or not because it's an error state?

Reply:
It *should never happen*™️, and I mainly put this in as a helper while I was building it, but yes better wrap it with a `t` while leaving it in